### DISCUSSION TITLE
Not Working for Past Couple of Updates?

### DISCUSSION THREAD

**Discussion by jycmba** [razaqq/PotatoAlert#47] [General]
Keep getting "connection refused" message and app requests to update but I've included it each time on Aslain mod updates, what am I missing??

**Comment by razaqq** [comment]
What region are you playing in?

**Reply by jycmba** [reply]
North America

**Comment by razaqq** [comment]
Aslain might not always have the latest version, so updating is in general very advisable

Can you join the discord server to try and debug the connection issue? https://discord.com/invite/Ut8t8PA
